specs/SetMemoryUsageLevel.md

```diff
@@ -0,0 +1,160 @@
+# Background
+Some WebView2 apps want to continue to run scripts while inactive and therefore cannot make usage of `TrySuspend` and `Resume` APIs to reduce resource consumption.
+We are introducing WebView2 API to reduce memory usage only for this type of inactive WebViews.
+
+# Description
```

Comment:
I'm not sure what this means to my app. Should I expect some scripts to run but others not? Or should I expect that some of my scripts will get out-of-memory errors?

Reply:
All scripts will continue to run. No impact to any scripts. If script runs after we swapped related memory out, we will swap the memory in. It is a best effort to reduce memory usage as much as possible.

Reply:
Please include something along these lines in the documentation 'If script runs after we swapped related memory out, we will swap the memory in. It is a best effort to reduce memory usage as much as possible.'

Reply:
Added explanation on what happens when script runs after memory is swapped out.